### PR TITLE
testgrid: Update generated OpenShift dashboards

### PR DESCRIPTION
**Pull request by smarterclayton**
Update to latest

/assign @bparees

### CODE REVIEW

File: config/testgrids/openshift/redhat-openshift-ocp-release-4.4-informing.yaml [hunk 1]
```diff
@@ -1433,9 +1406,6 @@ test_groups:
 - days_of_results: 60
   gcs_prefix: origin-ci-test/logs/release-openshift-ocp-installer-e2e-aws-ovn-4.4
   name: release-openshift-ocp-installer-e2e-aws-ovn-4.4
-- days_of_results: 60
-  gcs_prefix: origin-ci-test/logs/release-openshift-ocp-installer-e2e-aws-proxy-4.4
-  name: release-openshift-ocp-installer-e2e-aws-proxy-4.4
```
Comment (bparees) [inline config/testgrids/openshift/redhat-openshift-ocp-release-4.4-informing.yaml:1438]: we're losing the proxy job from informing?

Comment (smarterclayton) [reply config/testgrids/openshift/redhat-openshift-ocp-release-4.4-informing.yaml:1438]: Job was deleted by someone in openshift/release@9b767976ce295e875c20766a8bd60ba8537c033e

Comment (smarterclayton) [reply config/testgrids/openshift/redhat-openshift-ocp-release-4.4-informing.yaml:1438]: You should track with https://github.com/openshift/release/commit/9b767976ce295e875c20766a8bd60ba8537c033e#r40765657

**Comment by bparees** [comment]
/lgtm

will work with @ewolinetz  to get the new proxy job represented in testgrid.  that was something we discussed prior to landing his PR.

**Pull request merged**